test/cql-pytest/test_in_operator.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+from cassandra import InvalidRequest
+from cassandra.cluster import Session, NoHostAvailable
+from cassandra.protocol import SyntaxException
+
+from util import unique_name
+
+
+@pytest.fixture(scope="module")
+def table_name(request, cql: Session, test_keyspace: str) -> str:
+    table = f"{test_keyspace}.{unique_name()}"
+
+    cql.execute(f"CREATE TABLE {table} (p int PRIMARY KEY, a ascii, b boolean, d double,"
+                f"mai map<ascii, int>, tup frozen<tuple<text, int>>, l list<text>)")
+    request.addfinalizer(lambda: cql.execute("DROP TABLE " + table))
+    return table
+
+
+@pytest.mark.parametrize("column,column_type", [
+    ("mai", "map<ascii, int>"),
+    ("l", "list<text>"),
+])
```

Comment:
Nice use of parametrize(). I haven't used it yet in any test, and just used a loop, but the nice thing about parametrize is that when any of the cases fail, it automatically prints which one(s) failed.